doc/source/structures/orbits/orbitable.rst

```diff
@@ -201,22 +196,3 @@ All objects that can move in orbit around other objects share some similar struc
     :access: Get only
 
     The list of all the orbit patches that this object will transition to, not taking into account maneuver nodes. The zero-th patch of the list is the current orbit.
-
-.. attribute:: Orbitable:APOAPSIS
-
-    :type: :ref:`scalar <scalar>` (deg)
-    :access: Get only
-
-    .. deprecated:: 0.15
-    
-        Use :attr:`OBT:APOAPSIS <Orbit:APOAPSIS>` instead.
-
-.. attribute:: Orbitable:PERIAPSIS
-
-    :type: :ref:`scalar <scalar>` (deg)
-    :access: Get only
-
-    .. deprecated:: 0.15
-
-        Use :attr:`OBT:PERIAPSIS <Orbit:PERIAPSIS>` instead.
-
```

Comment:
Disagree (they're tied to the above mention which I disagreed with).

Reply:
Replaced.